solidity/test/signer.ts

```diff
@@ -0,0 +1,12 @@
+import type { Wallet } from 'ethers';
```

Comment:
Made this so we wouldn't need to //ts-ignore everywhere we get the signer

Reply:
Bummer, but definitely understand the burn and churn on getting hh-ethers to play nicely

Out of curiosity, what was preventing you from applying the hh types?